src/features/virtualized-table-renderers/__tests__/sizeCellRenderer.test.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+import getSize from '../../../utils/size';
+import sizeCellRenderer from '../sizeCellRenderer';
+
+jest.mock('../../../utils/size');
+
+describe('features/virtualized-table-renderers/sizeCellRenderer', () => {
+    let cellRendererParams;
+
+    beforeEach(() => {
+        cellRendererParams = {
+            cellData: 2000,
+        };
+        getSize.mockImplementation(value => `${value} petaflops`);
+    });
+
+    afterEach(() => {
+        jest.clearAllMocks();
+    });
+
+    test('should render a dash when cellData is missing', () => {
+        cellRendererParams.cellData = null;
+        expect(sizeCellRenderer()(cellRendererParams)).toBe('—');
+    });
+
+    test('should render a PrettyBytesCell', () => {
+        expect(sizeCellRenderer()(cellRendererParams)).toMatchSnapshot();
+    });
+
+    test('should set default multiplier when not provided', () => {
+        cellRendererParams.cellData = 1;
+        sizeCellRenderer()(cellRendererParams);
+        expect(getSize).toHaveBeenCalledTimes(1);
+        expect(getSize).toHaveBeenCalledWith(1);
```

Comment:
Minor: IMO, it's not immediately clear that `1` is the default multiplier, since both checks in this test take the same parameter. I recommend setting a constant for the default multiplier, i.e.,  `DEFAULT_MULTIPLIER = 1`.